<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanningViewControls</class>
 <widget class="QWidget" name="PlanningViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Awesome View</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbUseNode">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Use node for planning</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbShow2D">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Show 2D slices</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="lblThreshold">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderThreshold">
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Instrument 1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblInstrument">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vlPoints">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="pbLoadPointPlanning">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Load point planning</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbStartPointPlanning">
       <property name="text">
        <string>New point planning</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pbAddMeasurementPoint">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>220</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
